.github/workflows/pr_deploy.yaml

```diff
@@ -118,8 +104,9 @@ jobs:
           ref: ${{ needs.set-vars.outputs.ref }}
       - name: Release Comment on PR
         uses: marocchino/sticky-pull-request-comment@v2
-        if: ${{ github.event.inputs.pr-number }} != '' && success()
+        if: success()
         with:
+          append: true
```

Comment:
I'm hoping this is the magic needed so that when we do a rebase & push and then manually re-deploy, we'll have a way of knowing that the re-deploy wasn't forgotten and we're using the latest main code.